posts/2024-01-31-new-way-to-get-a-.gov.md

```diff
@@ -0,0 +1,54 @@
+---
+title: New way to get a .gov
```

Comment:
When looking at the list of blog posts, it doesn't immediately stand out that this is the announcing the launch of the request form. I wonder if there's a snappier title we an use?

Maybe something like: "We've launched a new way to get a .gov" ??

Reply:
You're so right. What about "Getting a .gov just got easier" or "Requesting a .gov domain just got easier"? @h-m-f-t What do you think?